### PR TITLE
fix: [PL-21050]: increase + icon on avatar groups

### PR DESCRIPTION
**Pull request by swarajcheguri-harness**
You can use the following comments to re-trigger PR Checks
image

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

File: src/components/Avatar/Avatar.tsx [hunk 1]
```diff
@@ -32,21 +32,59 @@ export interface AvatarProps extends HTMLDivProps {
   tooltip?: JSX.Element
   hoverCardDetailsCallBack?: (e: React.MouseEvent<HTMLDivElement>) => void
 }
+export const plusIcon = '+'
 interface SizeValueProps {
   size: string
   fontSize: string
   lineHeight: string
   imageHeight: string
+  plusIconFontSize: string
```
Comment (avinashmadhwani02) [inline src/components/Avatar/Avatar.tsx:41]: why are we taking this from props?

**Review by avinashmadhwani02 — CHANGES_REQUESTED** [review]

File: src/components/Avatar/Avatar.tsx [hunk 2]
```diff
@@ -32,21 +32,59 @@ export interface AvatarProps extends HTMLDivProps {
...
+  plusIconLineHeight: string
```
[the remark this answers is not in the document]
Comment (swarajcheguri-harness) [reply src/components/Avatar/Avatar.tsx:42]: in order make + icon to be kind of center aligned. Other ways to achieve this are appreciated

File: src/components/Avatar/Avatar.tsx [hunk 3]
```diff
@@ -32,21 +32,59 @@ export interface AvatarProps extends HTMLDivProps {
...
-  small: { size: '24px', fontSize: '10px', lineHeight: '18px', imageHeight: '20px' },
-  normal: { size: '32px', fontSize: '10px', lineHeight: '26px', imageHeight: '28px' },
-  medium: { size: '48px', fontSize: '18px', lineHeight: '42px', imageHeight: '44px' },
-  large: { size: '72px', fontSize: '28px', lineHeight: '66px', imageHeight: '68px' }
+  xsmall: {
+    size: '16px',
+    fontSize: '7px',
+    lineHeight: '12px',
+    imageHeight: '12px',
+    plusIconFontSize: '14px',
```
[the remark this answers is not in the document]
Comment (swarajcheguri-harness) [reply src/components/Avatar/Avatar.tsx:53]: want to keep the icon size open for change and not relative  to anything

File: src/components/Avatar/Avatar.tsx [hunk 4]
```diff
@@ -183,6 +221,13 @@ export const Avatar: React.FC<AvatarProps> = (props: AvatarProps) => {
   if (hoverCard) {
     hoverCardContent = { content: tooltip || defaultTooltip }
   }
+  let updatedContentStyle = {}
+  updatedContentStyle = { ...contentStyle }
+  if (initials === plusIcon) {
+    contentStyle.fontSize = sizes[size].plusIconFontSize
+    contentStyle.lineHeight = sizes[size].plusIconLineHeight
```
Comment (avinashmadhwani02) [inline src/components/Avatar/Avatar.tsx:228]: if we are anyways mutating contentStyle, why do we need `updatedContentStyle`?

**Review by avinashmadhwani02 — APPROVED** [review]

**Review by abhinavrastogi-harness — APPROVED** [review]

**Pull request merged**